backend/app/services/channels/dingtalk/handler.py

```
@@ -548,6 +898,282 @@ class ChatPayload:
+                        "💡 您可以在 Wegent 网页端查看执行进度和结果。"
+                    ),
+                    offset=0,
+                )
+                # Finalize the card to prevent it from staying in streaming state
+                await streaming_emitter.emit_chat_done(
+                    task_id=result.task.id,
+                    subtask_id=result.assistant_subtask.id,
+                    offset=0,
+                )
+
+            return None  # Success - no error message
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)